openapi/src/json-schemas/api/resources/answer-resource.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Answer resource",
+    "allOf": [
+        {"$ref": "../json-api-base-resource.json"},
+        {
+            "properties": {
+                "type": {
+                    "const": "answers"
+                }
```

Comment:
should we define a more specific `attributes` here and not just "inherit" the generic one?